<!-- -->
<launch>

  <arg name="frame_id_fix1" default="prius/odom"/>
  <arg name="frame_id_fix2" default="mkz/odom"/>
  <node pkg="cooperation_setup" type="fix_tf_publisher" name="fix_tf_publisher" output="screen">
    <param name="frame_id_fix1" value="$(arg frame_id_fix1)" />
    <param name="frame_id_fix2" value="$(arg frame_id_fix2)" />
    <remap from="fix1" to="prius/init/fix"/>
    <remap from="fix2" to="mkz/init/fix"/>
  </node>

</launch>
